### PR TITLE
WIP BXMSDOC-4568-master: Add a chapter about business central system properties in Configuring Business Central settings and properties doc.

### PR DESCRIPTION
**Pull request by kaldesai**
**Related Epic and JIRA:**
- [Epic](https://issues.jboss.org/browse/BXMSDOC-4405)
- [JIRA](https://issues.jboss.org/browse/BXMSDOC-4568)

**7.5 Rendered output links:**
- [Configuring Business Central settings and properties RHPAM doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4568-BC-SystemProperties-RHPAM/#business-central-system-properties-ref)
- [Configuring Business Central settings and properties RHDM doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4568-BC-SystemProperties-RHDM/#business-central-system-properties-ref)

**Community doc previews:**
- [Drools community doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4568-BC-SystemProperties-Drools/#business-central-system-properties-ref)
- [jBPM community doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4568-BC-SystemProperties-jBPM/#business-central-system-properties-ref)

Please review and verify **Chapter 19. Business Central system properties** from the RHPAM and RHDM docs. I have removed the domain mode part from the chapter.

### CODE REVIEW

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc [hunk 1]
```diff
@@ -66,13 +63,44 @@ If you plan to use RSA or any algorithm other than DSA, make sure you set up you
...
+
+* `kie.keystore.keyStoreURL`: The URL is used to load a Java Cryptography Extension KeyStore (JCEKS). For example, file:///home/kie/keystores/keystore.jceks.
+* `kie.keystore.keyStorePwd`: The password is used for the JCEKS.
+* `kie.keystore.key.ctrl.alias`: The alias of the key for default REST {CONTROLLER}.
+* `kie.keystore.key.ctrl.pwd`: The password of an alias for default REST {CONTROLLER}.
+* `org.jbpm.wb.forms.renderer.ext`: Switches the form rendering between {CENTRAL} and {KIE_SERVER}. By default, the form rendering is done by {CENTRAL}. Default value: `false`.
+* `org.jbpm.wb.forms.renderer.name`: This property allows to switch between {CENTRAL} and {KIE_SERVER} rendered forms. Default value: `workbench`.
++
+ifdef::DROOLS,JBPM[]
+To change one of these system properties in a *WildFly* or *JBoss EAP* cluster, use the following steps:
```
Comment (pefernan) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc]: I have doubts if we should keep this part. I'd like to Eder to take a look here.

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/Installation-section.adoc [hunk 2]
```diff
@@ -185,3 +69,5 @@ cn[\\ ]*=[\\ ]*role
 ----
 
 will find a match on role 'admin'.
+
+include::business-central-system-properties-ref.adoc[]
```
Comment (csherrar) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/Installation-section.adoc]: Is there the chance that this include could create duplication?

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc [hunk 3]
```diff
@@ -1,78 +1,91 @@
...
-* `org.kie.server.controller`: URL for connecting with a {CONTROLLER}, for example: `ws://localhost:8080/{URL_COMPONENT_CENTRAL}/websocket/controller`
+NOTE: If you plan to use RSA or any algorithm other than DSA, make sure you set up your application server to use the Bouncy Castle JCE library.
+
+* `org.uberfire.metadata.index.dir`: Place where the Lucene `.index` directory is stored. Default value: Working directory.
+* `org.uberfire.ldap.regex.role_mapper`: Regex pattern used to map LDAP principal names to the application role name. Note that the variable role must be part of the pattern because it is substituted by the application role name when matching a principal value to a role name.
+* `org.uberfire.sys.repo.monitor.disabled`: Disables the configuration monitor. Do not disable unless you are sure. Default value: `false`.
+* `org.uberfire.secure.key`: Password used by password encryption. Default value: `org.uberfire.admin`.
+* `org.uberfire.secure.alg`: Crypto algorithm used by password encryption. Default value: `PBEWithMD5AndDES`.
+* `org.uberfire.domain`:  Security-domain name used by uberfire. Default value: `ApplicationRealm`.
```
Comment (csherrar) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc]: There is a double space before "Security-domain".

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/Installation-section.adoc (hunk 2 above)
Comment (kaldesai) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/Installation-section.adoc]: This **include** is for the community documents.
All the community content is located in **Installation-section.adoc** file so there is no duplication.

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc (hunk 3 above)
Comment (kaldesai) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc]: Good catch!

**Pull request merged**